### ISSUE TITLE
UI: previous/next buttons overlap EPUB document scroll bar, and other interactive elements in fixed layout or reflowable publications (e.g. hyperlinks)

### ISSUE THREAD

**Issue by danielweck** [edrlab/readium-desktop#52]

**Comment by clebeaupin** [comment]
Could you provide an epub that embeds scrolling bar ?

**Comment by danielweck** [comment]
To test the scrollbar overlap, you can simply turn off ReadiumCSS pagination in the application's preferences (at which point EPUB reflowable HTML content documents will have a vertical scrollbar as per the default web browser behaviour CSS overflow:scroll).

**Comment by danielweck** [comment]
To test the overlap with other kinds of embedded content in both reflow and fixed-layout EPUBs, we could use Jiminy Panoz's "interaction" sample publication, which demonstrates forms, buttons, etc. for checking touch/mouse/keyboard reading system conflicts. This is in the readium-2 repository, but I cannot remember the exact location/branch.

**Comment by danielweck** [comment]
Note that every reading system have their own strategy for dealing with the left/right previous/next "clickable" areas. There is no golden standard. For example in Readium1 cloud reader and Chrome extension, the arrow buttons are intentionally placed outside of the content viewport area. On mobile devices with limited screen real estate, this is probably not the best layout, thus the overlap. Apple's iBooks does overlay the buttons but they also recommend content creators to avoid using this "margin-like" area for interactive content, and they fade-out the buttons when no mouse motion is detected (if I remember correctly).

**Comment by JayPanoz** [comment]
You’ll find it there, @danielweck → https://github.com/JayPanoz/readium-2/tree/master/misc/touch-handling (`touchHandling.epub`).

This should eventually be PR’ed in r2-glue-js at some point.

**Comment by danielweck** [comment]
Oh yes, awesome! :) Thanks Jiminy.
